docs/checks.md

```diff
@@ -161,11 +161,16 @@ which indicates that the project uses a set of security-focused best development
 source software. The check uses the URL for the Git repo and the OpenSSF Best Practices badge API.
 
 The OpenSSF Best Practices badge has 3 tiers: passing, silver, and gold. We give
-full credit to projects that meet the [passing criteria](https://bestpractices.coreinfrastructure.org/criteria/0), which is a
+full credit to projects that meet the [gold criteria](https://bestpractices.coreinfrastructure.org/criteria/1), which is a
```

Comment:
Oops, this should be `/2`, shall I update or would you like to (if you're doing some other magic to the commits)?

```suggestion
full credit to projects that meet the [gold criteria](https://bestpractices.coreinfrastructure.org/criteria/2), which is a
```